golang/internal/mapper/grpc.go

```diff
@@ -172,6 +176,20 @@ func mapCraneConfig(crane *agent.CraneContainerConfig, containerConfig *v1.Conta
 	if crane.ExtraLBAnnotations != nil {
 		containerConfig.ExtraLBAnnotations = crane.ExtraLBAnnotations
 	}
+
+	if crane.Labels != nil {
+		containerConfig.Labels = v1.Markers{
+			Deployment: crane.Labels.Deployment,
+			Service:    crane.Labels.Service,
+			Ingress:    crane.Labels.Ingress,
+		}
+
+		containerConfig.Annotations = v1.Markers{
```

Comment:
you missed to check annotations is nil or

Reply:
I've missed to add it to our tests, but I think it will fly still, with default values, gonna check.

Reply:
Nope, that was a mistake, nice catch.